packages/suite/src/components/suite/bluetooth/BluetoothDeviceComponent.tsx

```diff
@@ -27,13 +40,8 @@ export const BluetoothDeviceComponent = ({ device }: BluetoothDeviceProps) => {
 
     return (
         <Row gap={12} alignItems="stretch">
-            <Column alignItems="center" justifyContent="center">
-                <Image
-                    height="36px"
-                    imageSrc={resolveStaticPath(
-                        `images/png/t3w1/${internalModel.toLocaleLowerCase()}-back-color-${color}.webp`,
-                    )}
-                />
+            <Column alignItems="center" justifyContent="center" minWidth={22}>
```

Comment:
It is correct, the image displaes well. But I think it used to rotate.

But nevermind if this is correct image.